build.gradle.kts

```diff
@@ -141,7 +140,7 @@ dependencies {
 
 
   // Core lib, prefers local checkout if available
-  implementation(findProject(":core") ?: "org.veupathdb.lib:jaxrs-container-core:6.0.1")
```

Comment:
Are you sure?  This is frequently handy if you are working on a new feature in core....

Reply:
I did it when getting my local setup to work - I initially didn't understand that the core lib is wired up through what maven sees locally - I'll revert it!